### PR TITLE
Change math tool to CAT index tool in the workflow tutorial

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Change math tool to CAT index tool in the workflow tutorial

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@dbwiddis Could you review this PR when you get a chance? Thanks!

File: _automating-workflows/workflow-tutorial.md [hunk 1]
```diff
@@ -122,19 +122,18 @@ If you define `previous_node_inputs`, then defining edges is optional.
...
+- id: cat_index_tool
   type: create_tool
   user_inputs:
-    name: MathTool
-    type: MathTool
-    description: A general tool to calculate any math problem. The action input
-      must be a valid math expression, like 2+3
+    name: CatIndexTool
+    type: CatIndexTool
+    description: A tool to retrieve index information
```
Comment (dbwiddis) [inline _automating-workflows/workflow-tutorial.md:136]: I learned later that these descriptions are important to the quality of the tool's output and it's better to allow the default description.  So let's delete this line.  (Looks like you got all the others, just missed this one.)
```suggestion
```

**Review by dbwiddis — APPROVED** [review]
LGTM with one missed description deletion

File: _automating-workflows/workflow-tutorial.md [hunk 2]
```diff
@@ -143,15 +141,15 @@ You can configure other tools to be used by the CoT agent. For example, you can
 ### sub_agent
 <!-- vale on -->
 
-To use the math tool in the agent configuration, specify it as one of the tools in the `previous_node_inputs` field of the agent. You can add other tools to `previous_node_inputs` as necessary. The agent also needs a large language model (LLM) in order to reason with the tools. The LLM is defined by the `llm.model_id` field. This example assumes that the `model_id` from the `deploy_model_3` step will be used. However, if another model is already deployed, the `model_id` of that previously deployed model could be included in the `user_inputs` field instead:
+To use the `cat_index_tool` in the agent configuration, specify it as one of the tools in the `previous_node_inputs` field of the agent. You can add other tools to `previous_node_inputs` as necessary. The agent also needs a large language model (LLM) in order to reason with the tools. The LLM is defined by the `llm.model_id` field. This example assumes that the `model_id` from the `deploy_model_3` step will be used. However, if another model is already deployed, the `model_id` of that previously deployed model could be included in the `user_inputs` field instead:
```
Comment (Naarcha-AWS) [inline _automating-workflows/workflow-tutorial.md:144]: What do we mean by "to reason with the tools"?

**Review by Naarcha-AWS — APPROVED** [review]
Quick clarifying question, but otherwise LGTM.

File: _automating-workflows/workflow-tutorial.md (hunk 2 above)
Comment (kolchfa-aws) [reply _automating-workflows/workflow-tutorial.md:144]: The LLM uses the knowledge base and the tools for reasoning.

**Pull request merged**